src/credentials/CHIPCert.h

```diff
@@ -265,6 +267,16 @@ class ChipDN
      **/
     CHIP_ERROR GetCertFabricId(uint64_t & fabricId) const;
 
+    /**
+     * @brief Decode ChipDN attributes from TLV encoded format.
```

Comment:
Needs to document where the reader is expected to be positioned: on the TLV list element that encodes the DN.

Reply:
added